### PR TITLE
[Scrutinizer] App.php / Create Model\Process

### PR DESCRIPTION
**Pull request by MrPetovan**
Part of #4176

This is tested on my node without errors.

### CODE REVIEW

File: src/App.php [hunk 1]
```diff
@@ -518,67 +509,70 @@ function init_pagehead() {
...
 		$this->curl_headers = $headers;
 	}
 
-	function get_curl_headers() {
+	public function get_curl_headers()
+	{
 		return $this->curl_headers;
 	}
 
-	function get_cached_avatar_image($avatar_image) {
```
Comment (annando) [inline src/App.php]: This function is in use in two addons.

Comment (MrPetovan) [reply src/App.php]: Yeah, I forgot to submit the addons PR.

Comment (annando) [reply src/App.php]: Reminder ;-)

Comment (MrPetovan) [reply src/App.php]: Apparently I didn't commit anything yesterday, so this will have to wait tonight.

**Comment by MrPetovan** [comment]
PR submitted to addon: https://github.com/friendica/friendica-addons/pull/488

File: src/App.php (hunk 1 above)
Comment (MrPetovan) [reply src/App.php]: Done.

**Pull request merged**